### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by aeschweik**
Raised because of confusion over nature of errors

### CODE REVIEW

File: vrecord [hunk 1]
```diff
@@ -1338,14 +1338,14 @@ if [[ "${FRAMEMD5_CHOICE}" = "Yes" ]] ; then
         _writeingestlog "PTS_DISCONTINUITY" "none"
     else
         _writeingestlog "PTS_DISCONTINUITY" "${PTS_DISCONTINUITY}"
-        cowsay "$(_report -w "WARNING: There were pts discontinuities for these frame ranges: ${PTS_DISCONTINUITY}. The file may have sync issues.")"
+        cowsay "$(_report -w "WARNING: There were discontinuities in the file's frame MD5s for these frame ranges: ${PTS_DISCONTINUITY}. This error may indicate frames dropped by FFmpeg or vrecord. The file may have sync issues.")"
```
Comment (dericed) [inline vrecord]: I think removing the reference to pts here changes the meaning. The discontinuities are not in the md5s but in the temporal differences between the sequential pts from the frame md5 report. You could say something like "There were presentation timestamp discontinuities found in the framemd5 for ..."

File: README.md [hunk 2]
```diff
@@ -270,10 +270,10 @@ If you are finished recording and the player window hasn't already closed, close
 After the transfer is finished, vrecord will automatically check for the following transfer errors:
 
 * Discontinuities in the frame MD5s (if they were created), or missing frames in the FFmpeg log (if frame MD5s were not created).
-  * Error message: "WARNING: There were pts discontinuities for these frame ranges: ##-##. The file may have sync issues" or "WARNING: FFmpeg reported missing frames. The file may have sync issues." The message may give the frame numbers that are missing. Check the file immediately at these points and throughout the video to make sure there are no sync issues.
+  * Error message: "WARNING: There were discontinuities in the file's frame MD5s. This error may indicate frames dropped by FFmpeg or vrecord. The file may have sync issues." The message may give the frame numbers that are missing. Check the file immediately at these points and throughout the video to make sure there are no sync issues.
```
Comment (dericed) [inline README.md]: same comment as on line 1341 of vrecord

File: vrecord (hunk 1 above)
Comment (retokromer) [reply vrecord]: Good point!

File: README.md [hunk 3]
```diff
@@ -269,11 +269,11 @@ If you are finished recording and the player window hasn't already closed, close
 
 After the transfer is finished, vrecord will automatically check for the following transfer errors:
 
-* Discontinuities in the frame MD5s (if they were created), or missing frames in the FFmpeg log (if frame MD5s were not created).
-  * Error message: "WARNING: There were pts discontinuities for these frame ranges: ##-##. The file may have sync issues" or "WARNING: FFmpeg reported missing frames. The file may have sync issues." The message may give the frame numbers that are missing. Check the file immediately at these points and throughout the video to make sure there are no sync issues.
+* Presentation timestamp discontinuities in the frame MD5s (if they were created), or missing frames in the FFmpeg log (if frame MD5s were not created).
+  * Error message: "WARNING: There were presentation timestamp discontinuities found in the framemd5s. This error may indicate frames dropped by FFmpeg or vrecord. This error may indicate frames dropped by FFmpeg or vrecord. The file may have sync issues." The message may give the frame numbers that are missing. Check the file immediately at these points and throughout the video to make sure there are no sync issues.
```
Comment (retokromer) [inline README.md]: The sentence `This error may indicate frames dropped by FFmpeg or vrecord.` is present twice.

**Pull request merged**